### DISCUSSION TITLE
Adaptive Moving Average

### DISCUSSION THREAD

**Discussion by bluefish0125** [nautechsystems/nautilus_trader#1359] [Q&A] [answered]
So I been working through the quick start documentation and I wanted to try out the built in adaptive moving average indicator. However, on the Nautilus Trader page, I cannot find what to import. For the macd we would import MovingAverageConvergenceDivergence from nautilus_trader.indicators.macd. What should we import for the AMA?

**Comment by cjdsellers** [comment, marked as answer]
Hey @bluefish0125 

You could try this:
```
from nautilus_trader.indicators.average.ama import AdaptiveMovingAverage
```

**Reply by bluefish0125** [reply]
Thank you so much! Was wondering where on the website we could find more information on modules and their imports?

**Reply by cjdsellers** [reply]
The API reference is still your best bet https://docs.nautilustrader.io/api_reference/index.html
Just be sure to select either `latest` for latest release, or `develop` if you're compiling from source.

**Reply by bluefish0125** [reply]
Great thank you!
